Nonlinear_Model_Predictive_for_WECCCOMP/wecSimInputFile.m

```diff
@@ -0,0 +1,153 @@
+%% WECCCOMP model of WaveStar device with WAMIT data 
+% https://github.com/WEC-Sim/WECCCOMP
+%Select the Sea State to simulate
+SeaState = 6;
+switch(SeaState)
+    case 1 ;        Hm0=0.0208;        Tp=0.988;        gamma=1.0;
+    case 2 ;        Hm0=0.0625;        Tp=1.412;        gamma=1.0;
+    case 3 ;        Hm0=0.1042;        Tp=1.836;        gamma=1.0;
+    case 4 ;        Hm0=0.0208;        Tp=0.988;        gamma=3.3;
+    case 5 ;        Hm0=0.0625;        Tp=1.412;        gamma=3.3;
+    case 6 ;        Hm0=0.1042;        Tp=1.836;        gamma=3.3;
+end
+
+%% Simulation Class
+simu = simulationClass();                           % Create the Simulation Variable
+    simu.simMechanicsFile = 'WaveStar.slx';         % Specify Simulink Model File       
+    simu.dt             = 50/1000;                  % Simulation Time-Step [s]
+    simu.rampTime       = 25;                       % Wave Ramp Time Length [s]
+    switch(SeaState)
+        case { 1, 4 };        simu.endTime        = 100 + simu.rampTime;
+        case { 2, 5 };        simu.endTime        = 150 + simu.rampTime;
+        case { 3, 6 };        simu.endTime        = 200 + simu.rampTime;
+    end
+    simu.CITime         = [];                        % Convolution Time [s]
+    simu.explorer       = 'off';                     % Explorer on
+    simu.solver         = 'ode8';                   % Turn on ode45

```

Comment:
@Liam-Guerrero Is there a particular reason why you needed to use `ode8`? Although WEC-Sim should be able to use any fixed- or variable-time step solver we generally suggest `ode4` or `ode45` per the comment on this line.

Reply:
Yes, the reason for using this solver is because it was the only one that allows running the model at a higher sampling rate. The original version of WECCCOMP runs under 10 ms, but I need it to be at 50 ms. 

I read the proposed solution you offer in Issue #582, but the simulation crashed even after implementing that solution.